src/__tests__/reporter/testIemReporting.spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import MyReporter from '../../reporter';
+import { mockConfig } from '../mocks/configMock';
+import { mockedResult, mockedTestParams, RPClientMock } from '../mocks/RPClientMock';
+import { FinishTestItemObjType, StartTestObjType } from '../../models';
+import { TEST_ITEM_TYPES } from '../../constants';
+import * as utils from '../../utils';
+
+describe('start test reporting', () => {
```

Comment:
This `describe` block and the previous (from file `suitesReporting.spec.ts`) test the same reporter's method.
As the suites and test items reported from one method it would be better to hold them in one file under one `describe` block.